cmd/lint.go

```diff
@@ -299,7 +299,7 @@ func lint(args []string, params *lintCommandParams) (report.Report, error) {
 		}
 
 		regal = regal.WithUserConfig(userConfig)
-	case err != nil && params.configFile != "":
+	case params.configFile != "":
```

Comment:
This is unrelated. Just fixing a redundant check as the condition will always be true.